.circleci/config.yml

```diff
@@ -73,7 +73,7 @@ jobs:
           name: Tag the Docker image and push it to Google Container Registry.
           command: |
             if [ -n "$CIRCLE_TAG" ]; then
-              export MAJOR=`echo $CIRCLE_TAG | awk -F '.' '{ print $1; }'`
+              export MAJOR=`echo ${CIRCLE_TAG:1} | awk -F '.' '{ print $1; }'`
```

Comment:
Can you add this to the MINOR & PATCH tags below as well?

Reply:
sure